### PR TITLE
Dev mariateo proud section finished

### PR DESCRIPTION
**Pull request by MariaTeo**

### CODE REVIEW

File: src/_component/proud/proud.pug [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+          li.lista
+            .texte
+              .date=item.date
+              .award=item.award
+              .location=item.location
+.column(span="row" height="10")
+.column(span="row")
+  button.button__disc--green
+    i.icon__arrow-down
+.column(span="row" height="10")
```
Comment (RalucaRbc) [inline src/_component/proud/proud.pug:21]: + 1 empty row at the and of file

File: src/_component/proud/proud.scss [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  margin-right: 10px;
+}
+
+.box__reviews {
+  border-top: 0;
+}
+
+.texte {
+  @extend %list-item;
+}
```
Comment (RalucaRbc) [inline src/_component/proud/proud.scss:35]: +1 empty row here at the end

File: src/index.pug [hunk 3]
```diff
@@ -15,3 +15,4 @@ html
         include _component/portfolio/portfolio.pug
         include _component/recommendations/recommendations.pug
         include _component/reviews/reviews.pug
+        include _component/proud/proud.pug
```
Comment (RalucaRbc) [inline src/index.pug:18]: +1 empty row here

File: src/style.scss [hunk 4]
```diff
@@ -10,4 +10,5 @@
 @import '_component/portfolio/portfolio.scss';
 @import '_component/recommendations/recommendations.scss';
 @import '_component/reviews/reviews.scss';
+@import '_component/proud/proud.scss';
 @import '_asset/font/style.scss'
```
Comment (RalucaRbc) [inline src/style.scss:14]: +1 empty row here

**Review by Constantin-Matei — APPROVED** [review]
:)

**Review by RalucaRbc — APPROVED** [review]
👍

**Review by samuray90 — APPROVED** [review]

**Review by Adrian01C — COMMENTED** [review]
cool

**Comment by Adrian01C** [comment]
cool

File: src/_component/proud/proud.pug (hunk 1 above)
Comment (MariaTeo) [reply src/_component/proud/proud.pug:21]: The line is there ;)

File: src/_component/proud/proud.scss (hunk 2 above)
Comment (MariaTeo) [reply src/_component/proud/proud.scss:35]: the line is there ;)

**Pull request merged**